### ISSUE TITLE
Split covid19 dataset group by patient ID

### ISSUE THREAD

**Issue by oplatek** [mlmed/torchxrayvision#2]
The `COVID19_Dataset(Dataset)` provides image and binary decision is_covid but lacks patient ID.

**In the process of creating the dataset, it drops the information about the patient ID.**
https://github.com/mlmed/torchxrayvision/blob/master/torchxrayvision/datasets.py#L859

**Why it matters?**
I would like to avoid having images from the same patient in both train and valid (or test set)?

**How to solve it?**
Could you please provide one of the following:
-  the patient ID on the output
- or provide a split method which respects the patient ID?
Thanks!

**Comment by oplatek** [comment]
See https://github.com/PyTorchLightning/lightning-Covid19/issues/11

**Comment by ieee8023** [comment]
Sorry for being so late on this reply! The patientid should have been there but our documentation was not the best. That dataset does have a `.csv.patientid` field. Here is how it can be achieved for most of the datasets in the library:

<img width="720" alt="Screen Shot 2022-01-30 at 19 49 53" src="https://user-images.githubusercontent.com/446367/151736302-56efd6c5-77ed-48a9-a386-e41733ed0b3f.png">
